test/unittest/xmlparser/wrapper/XMLParserTest.hpp

```diff
@@ -252,10 +252,18 @@ class XMLParserTest : public XMLParser
 
     static XMLP_ret getXMLThroughputController_wrapper(
             tinyxml2::XMLElement* elem,
-            ThroughputControllerDescriptor& throughputController,
+            ThroughputControllerDescriptor& throughtController,
             uint8_t ident)
     {
-        return getXMLThroughputController(elem, throughputController, ident);
+        return getXMLThroughputController(elem, throughtController, ident);
```

Comment:
I think this was not intended to be changed
```suggestion
        return getXMLThroughputController(elem, throughtputController, ident);
```